app/src/main/java/com/mapbox/maps/testapp/examples/annotation/PolylineAnnotationActivity.kt

```diff
@@ -101,7 +103,12 @@ class PolylineAnnotationActivity : AppCompatActivity() {
       }
     }
 
-    deleteAll.setOnClickListener { lineManager?.deleteAll() }
+    deleteAll.setOnClickListener {
+      lineManager?.let {
+        it.deleteAll()
```

Comment:
is there a reason why we need to delete all annotations?

Reply:
No, we can delete this line.